src/main/java/edu/harvard/iq/dataverse/settings/JvmSettings.java

```diff
@@ -123,6 +123,10 @@ public enum JvmSettings {
     SCOPE_UI(PREFIX, "ui"),
     UI_ALLOW_REVIEW_INCOMPLETE(SCOPE_UI, "allow-review-for-incomplete"),
     UI_SHOW_VALIDITY_FILTER(SCOPE_UI, "show-validity-filter"),
+
+    // NetCDF SETTINGS
+    SCOPE_NETCDF(PREFIX, "netcdf"),
```

Comment:
Can we please start having an "ingest" scope where we collect these?

Reply:
Maybe? The timing is poor because I was away for two weeks and now you're away until August.

What are you suggesting, specifically?

Instead of...

`dataverse.netcdf.geo-extract-s3-direct-upload`

... you'd like to see...

`dataverse.ingest.netcdf.geo-extract-s3-direct-upload`?

I'm ok with it if @landreev and/or @scolapasta are ok with it. And @sekmiller did the review. And @qqmyers does a lot of storage work. They might want to weigh in.

For the record, I looked for ingest stuff in our config page and we do already have these:

- `dataverse.files.<id>.ingestsizelimit`
- `:TabularIngestSizeLimit`

Personally, I've been referring to this feature as "metadata extraction" (from a file) rather than ingest. I'm not sure where we draw the line on what ingest is.

I'll plug having a glossary, like I always do! 😜 

- #4274